app/scripts/lib/crypto/recovery-keys.js

```diff
@@ -0,0 +1,175 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  const sjcl = require('sjcl');
+
+  function importFxaCryptoDeriver() {
+    return import(/* webpackChunkName: "fxaCryptoDeriver" */ 'fxaCryptoDeriver');
+  }
+
+  const RECOVERY_KEY_VERSION = '\x01';
+  const ALPHABET_BASE32 = '0123456789ABCDEFGHJKMNPQRSTVWXYZ';
```

Comment:
I should have checked this earlier, but I see this alphabet differs from [RFC 3548 § 5](https://tools.ietf.org/html/rfc3548#section-5).  Is that a concern?

Reply:
I believe this was implementation the sjcl base32 used for encoding/decoding. It might be a non issue if I use the `hi-base32` module (which I believe can decode any version).